Doc/library/shutil.rst

```diff
@@ -74,6 +76,10 @@ Directory and files operations
       Raise :exc:`SameFileError` instead of :exc:`Error`.  Since the former is
       a subclass of the latter, this change is backward compatible.
 
+   .. versionchanged:: 3.8
+      Platform-specific zero-copy syscalls are used internally in order to copy
+      the file more efficiently. See
+      `platform-dependent efficient copy operations`_ section.
```

Comment:
Since we're in Sphinx rather than plain `docutils`, this can be :ref:`shutil-platform-dependent-efficient-copy-operations`.

I'd also be inclined to shorten the initial sentence even further. Something like "Optimised platform-specific calls are used to copy files more efficiently. See ... for more details."